### ISSUE TITLE
rails_helper.rb is not loaded when running the RunNearestSpec.

### ISSUE THREAD

**Issue by NafaaBout** [thoughtbot/vim-rspec#78]
when I run the RunNearestSpec using the mapped keys, the `rails_helper` is not loaded, which makes some problems for me. Since the configuration for the database cleaner gem is added to that file, it is not loaded and thus the database cleaner does not clean my database. 
This problem does not exists with the other commands, like `RunCurrentSpecFile`, and `RunAllSpecs`

**Comment by gylaz** [comment]
Can you tell what command is being executed when you `RunNearestSpec` is execute? It should be something like `rspec spec/models/some_model_spec.rb:11`. Can you run just that command from vim? If not, there's something up with your vim, or Rails setup. This plugin knows nothing about spec helpers, it just runs commands.

**Comment by MatthewMDavis** [comment]
I just wrestled with the problem for the last several hours. Here's what happens: vim switches its current working directory to the directory where the spec file is located. If you were still in the Rails project root directory, it would find rails_helper.rb. If you cd to the Rails project root, and THEN fire off your rspec commands, they should work. I haven't figured out a good way around the problem yet; I just figured out what it is.
